src/einsteinpy/symbolic/q

```diff
@@ -0,0 +1,458 @@
+import numpy as np
```

Comment:
And why all these new files?

Reply:
whoops! looks like I accidentally git added unwanted files. I'll remove them. sorry.